distributed/protocol/utils.py

```diff
@@ -23,21 +23,15 @@ def frame_split_size(frame, n=BIG_BYTES_SHARD_SIZE) -> list:
     >>> frame_split_size([b'12345', b'678'], n=3)  # doctest: +SKIP
     [b'123', b'45', b'678']
     """
-    if nbytes(frame) <= n:
+    frame = memoryview(frame)
```

Comment:
Are we always guaranteed that will pass ?  Previously we had a try/except

Reply:
Yeah

The previous `try...except...` was guarding against the absence of an `itemsize` attribute. Though `memoryview` always has this. So that is not an issue.